css-two-column-layout/styles.css

```diff
@@ -0,0 +1,82 @@
+.col-full {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.header {
+  max-width: 600px;
+  text-align: center;
+  line-height: 2;
+}
+
+.container-2 {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: auto;
+  max-width: 960px;
+}
+
+.col-half {
+  display: flex;
+  justify-content: center;
+  align-items: center;
```

Comment:
Don't pollute your layout classes. Your `column` classes' rulesets should only have `flex-basis` or `width` declarations, and nothing else. If you are making a `column` class into `display: flex`, this is an indicator you should have a `row` inside this `column`.